### ISSUE TITLE
Buffering issue's 

### ISSUE THREAD

**Issue by hamswolo** [yuliskov/SmartTubeNext#704]
Still having random buffering issues and random can't load video message .I have 3 firesticks doing it on all of them . Not nearly as bad as before but still occurring . I can usually stop the app reload it but today one video will buffer no matter what I did .

**Comment by Gertjan77** [comment]
Yeah the problem is back today for me as well. Last couple of days everything worked fine.

**Comment by Grauen** [comment]
https://github.com/yuliskov/SmartTubeNext/issues/658

**Comment by heliosphann** [comment]
Still have buffering issues on my Nvidia Shield after the 11.42 update.

Can close app and it work work for a while, but buffering starts up again.

**Comment by Grauen** [comment]
After 11.45 it looks better

**Comment by BalloS-Twok** [comment]
Still with 11.60 on fire TV Stick.
Seems to be worse than preversion

**Comment by smogan** [comment]
Here the Same on Fire TV 4K Stick and Version 11.65 and all the last Version before. Problem seems random. Sometimes no Problem at all, even for a whole Day. but when it occurs it stays for a long while. I tried everything, I even reset the stick to factory default one time. Occurs on both Amazon Sticks Here. 

Seems Fire TV specific, cause on my Xiaomi Mi Stick I never experienced buffering before.

**Comment by BCshield** [comment]
Having the same issue intermittently with my shield.  I have noticed that it seems to happen when my son starts using Discord.  Could there be some conflict?
